mitm.go

```diff
@@ -178,6 +186,11 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	p.proxyMITM(sc, cc)
 }
 
+// By default we don't skip any request and proxy all of them.
```

Comment:
Super-nitpicky, but the convention is to always start the doc with the function name (so something like "SkipNone indicates that no requests should be skipped." or something like that).